jcl/source/windows/JclDebug.pas

```diff
@@ -3498,6 +3545,73 @@ function TJclBinDebugScanner.SourceNameFromAddr(Addr: DWORD): string;
     Result := '';
 end;
 
+function TJclBinDebugScanner.VAFromUnitAndProcName( UnitName, ProcName : string ): DWORD;
+var
+  P          : Pointer;
+  VA         : Cardinal;
+  i,
+  Value,
+  FirstWord,
+  SecondWord : Integer;
+  S          : string;
+begin
+  Result := 0;
+  if ( UnitName = '' ) or ( ProcName = '' ) then
```

Comment:
No spaces inside parenthesis, also applies to all the other lines in this method